api_tests/users/views/test_user_actions.py

```diff
@@ -18,11 +18,13 @@ class TestReviewActionFilters(ReviewActionFilterMixin):
     def url(self):
         return '/{}actions/reviews/'.format(API_BASE)
 
+    def get_expected_actions(self, all_actions, allowed_providers):
```

Comment:
Why was this redefined from api_tests/reviews/mixins/filter_mixins.py

Reply:
Pytest 5.0.0 fully depreciated callable fixtures, so it threw and error and failed in pytest 5